tests/lib/state_test.py

```diff
@@ -882,3 +883,76 @@ def test_match_nested_dict_in_list(self):
             [{'a': 2, 'b': [3, 4]}, {'a': 1, 'b': [1, 2]}],
             [{'a': 1, 'b': [1, 2]}, {'a': 2, 'b': [3, 4]}],
         )
+
+
+class TestRouteRuleEntry():
```

Comment:
CI complains here: https://travis-ci.com/nmstate/nmstate/jobs/261874110#L866
You need to drop the `()`.